lib/composers/closeComposer.js

```diff
@@ -0,0 +1,12 @@
+const {makeModifierComposer} = require('../utils/makeComposer');
+const composers = require('../constants/composer');
+
+/**
+ * Defined close method
+ */
+const closeComposer = makeModifierComposer(composers.CLOSE_APP, ['close'], (_, meta) => ({
+	...meta,
+	isClosed: true,
```

Comment:
`isClosed` might be a bit confusing, without looking into context too deep I would assume that this is a check for the app being closed. I.e. like "application().hasExited()".

`isCloseApp` maybe?